lib/styles/settings/typography/_tokens.json

```diff
@@ -0,0 +1,12245 @@
+{
+  "DisplayHeading": [
```

Comment:
does our parser prettify json output, or will I get this file updated when I synchronize token again?

Reply:
It was manual reformat by me, I reverted it.